src/App.tsx

```diff
@@ -1,17 +1,122 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import './App.scss';
+import TodoList from './components/TodoList/TodoList';
 
 import users from './api/users';
+import todos from './api/todos';
+
+interface Todo {
+  userId: number;
+  id: number;
+  title: string;
+  completed: boolean;
+  userLink?: {
+    name: string;
+    email: string;
+  }
+}
```

Comment:
move all types to a separate file

Reply:
and export them from there, so you can use them in multiple files

Reply:
ok thx I will do it!